cookbook/core/type_system/structured_dataset.py

```diff
@@ -50,8 +50,8 @@
 
 
 # %%
-# We define two tasks — one returns a pandas DataFrame, and the other, a :py:class:`flytekit.types.schema.FlyteSchema`.
-# The DataFrames themselves will be serialized to an intermediate format like parquet before sending them to the other tasks.
+# We define two tasks— one returns a pandas DataFrame, and the other, a :py:class:`flytekit.types.schema.FlyteSchema`.
```

Comment:
Suggestion: "We define two tasks, one returns a pandas DataFrame and the other a". Rationale: don't need that many commas. :)